nlp_architect/models/tagging.py

```diff
@@ -202,13 +219,16 @@ def train(
         save_steps: int = 100,
         save_path: str = None,
         distiller: TeacherStudentDistill = None,
+        best_result_file: str = None,
```

Comment:
why store the best result here? this should be stored as part of the training script